ddtrace/llmobs/_integrations/google_utils.py

```diff
@@ -184,16 +187,25 @@ def extract_message_from_part_google_genai(part, role: str) -> Dict[str, Any]:
 
     function_call = _get_attr(part, "function_call", None)
     if function_call:
-        function_name = _get_attr(function_call, "name", "")
-        function_args = _get_attr(function_call, "args", {})
-        message["tool_calls"] = [{"name": str(function_name), "arguments": function_args}]
+        tool_call_info = ToolCall(
+            name=_get_attr(function_call, "name", ""),
+            arguments=_get_attr(function_call, "args", {}),
```

Comment:
Similar comment here... should we make sure we are not returning None for these fields since they are optional in the `function_call`?
```suggestion
            name=_get_attr(function_call, "name", "") or "",
            arguments=_get_attr(function_call, "args", {}) or {},
```